src/hotspot/os/windows/gc/x/xMapper_windows.cpp

```diff
@@ -115,13 +115,13 @@ HANDLE XMapper::create_paging_file_mapping(size_t size) {
 bool XMapper::commit_paging_file_mapping(HANDLE file_handle, uintptr_t file_offset, size_t size) {
   const uintptr_t addr = map_view_no_placeholder(file_handle, file_offset, size);
   if (addr == 0) {
-    log_error(gc)("Failed to map view of paging file mapping (%d)", GetLastError());
+    log_error(gc)("Failed to map view of paging file mapping (%ld)", GetLastError());
```

Comment:
Here, and in many other places:

All these `%d` -> `%ld` changes are questionable. Why do you think `l` is more correct?

Strictly spoken both existing and your versions are incorrect since GetLastError returns a DWORD, which is an unsigned 32-bit integer.

Were I to change anything, I would print an unsigned integer. But I would not change anything, since all documented error codes are well below 0x8000_0000. I think we are good here.

Reply:
DWORD is defined as an unsigned long on all Windows compilers, which more accurately maps to %ld under C++ rules. This was more for correctness, but may not be strictly needed